internal/virter/dhcp.go

```diff
@@ -93,6 +65,45 @@ func qemuMAC(id uint) string {
 	return fmt.Sprintf("52:54:00:%02x:%02x:%02x", id0, id1, id2)
 }
 
+func (v *Virter) getIPNet(network libvirt.Network) (net.IPNet, error) {
+	ipNet := net.IPNet{}
+
+	networkDescription, err := getNetworkDescription(v.libvirt, network)
+	if err != nil {
+		return ipNet, err
+	}
+	if len(networkDescription.IPs) < 1 {
+		return ipNet, fmt.Errorf("no IPs in network")
```

Comment:
very minor, but all the `fmt.Errorf` in this function could be `errors.New`

Reply:
Good to know. I think I'll leave it because it just makes the code more consistent if we use `fmt.Errorf` everywhere.